tdrs-backend/plg/grafana_views/user_views/fra_te1.sql

```diff
@@ -0,0 +1,48 @@
+-- AUTOMATICALLY GENERATED FILE ON 2025-05-08 08:27:24
+-- DO NOT EDIT - Your changes will be overwritten
+-- Generated by generate_user_views.py
+
+-- SQL view for fra_te1 schema
+-- Transformations applied:
+--   * SSN transformed to md5 hash for privacy
+
+
+CREATE OR REPLACE VIEW "fra_te1" AS SELECT "EXIT_DATE",md5("SSN"::text) as "SSN_HASH",CASE
```

Comment:
i think its okay to keep this for now, but not sure how we will use it since we need the real identifiers for the match. for now we'll plan to use the admin version.